xbmc/music/tags/MusicInfoTag.cpp

```diff
@@ -381,10 +381,10 @@ void CMusicInfoTag::SetAlbumArtist(const std::string& strAlbumArtist)
   }
 }
 
-void CMusicInfoTag::SetAlbumArtist(const std::vector<std::string>& albumArtists)
+void CMusicInfoTag::SetAlbumArtist(const std::vector<std::string>& albumArtists, bool FillDesc /* = false*/)
 {
   m_albumArtist = albumArtists;
-  if (m_strAlbumArtistDesc.empty()) 
+  if (m_strAlbumArtistDesc.empty() || FillDesc) 
```

Comment:
Tested the build.  Tests passed.  Works per requirements (files containing both ID3v2.3 and APEv2 tags use the ID3 tags unless advancedsetting prioritiseapetags is set.  In that case APE tag is used in preference to ID3 tag.)